docs/ops/doc/examples/deconvolution.rst

```diff
@@ -80,7 +80,7 @@ SciJava Ops via Fiji's scripting engine with `script parameters`_:
         pZ = pZ * 1E-6
 
         // create the synthetic PSF
-        psf = ops.op("create.kernelDiffraction").arity9().input(psf_size,
+        psf = ops.op("create.kernelDiffraction").input(psf_size,
                                                                 numericalAperture,
```

Comment:
These parameters are no longer indented in alignment with the first parameter `psf_size`.